app/default/scripts/cameras_calib.xml.template

```diff
@@ -4,7 +4,12 @@
 </dependencies> 
 <module>
   <name>yarpdev</name>
-      <parameters> --from camera/ServerGrabberDualDragon.ini --split true</parameters>
+      <parameters> --from camera/dragonfly2_config_left.ini --split true</parameters>
```

Comment:
The parameter `--split true` should be only meaningful with the dual driver.
@Nicogene, could you confirm this?
If confirmed, we ought to remove it.

Reply:
Yes I confirm it is obsolete 👍🏻